linux_os/guide/services/base/service_kdump_disabled/policy/stig/shared.yml

```diff
@@ -0,0 +1,45 @@
+srg_requirement: |-
```

Comment:
Why does the content of the policy file appear in the resolved rule yaml when I build a different product than RHEL 9, for example RHEL 8, for example I can see this text in `policy_specific_content` key `build/rhel8/rules/service_kdump_disabled.yml`. Is that expected behavior or not?

Reply:
It is. Since the content is in the `shared.yml` file all products will get the content. Products can override this by adding their own files. For example, `rhel8.yml`. Currently, family files (e.g., `rhel.yml`) are not supported.